docs/index.md

```diff
@@ -0,0 +1,58 @@
+# nbgitpuller
+
+`nbgitpuller` lets you distribute content to a Jupyter user via the click of a button!
+This allows your users to focus on the content without needing to understand `git`
+or other version control machinery.
+
+`nbgitpuller` provides {ref}`automatic, opinioned conflict resolution <topic/automatic-merging>`
+by using `git` under the hood.
+It is commonly used to distribute content to multiple users of a JupyterHub, though it works just fine on an individual person's computer, if they have Jupyter installed.
+
+Here's an example of `nbgitpuller` in action:
+
+```{image} _static/nbpuller.gif
+```
+
+## Use `nbgitpuller`
+
+See [](use.md) for information about how to use `nbgitpuller`.
+Here's a short overview:
+
+1. Ensure your user has `nbgitpuller` installed. This is commonly done by installing it for all users of a JupyterHub (or, you may ask them to install it manually). See [the installation instructions](install.rst) for more information.
+2. Create an "`nbgitpuller` link" which points to the content you'd like to distribute (e.g., a Jupyter Notebook in a GitHub repository).
+3. Tell your user to click the link, and `nbgitpuller` will automatically pull in the content to their file system.
+
+```{admonition} Try the nbgitpuller link generator!
```

Comment:
Can we point them to the Browser extensions here as well? https://github.com/yuvipanda/nbgitpuller-link-generator-webextension.

Chrome: https://chrome.google.com/webstore/detail/nbgitpuller-link-generato/hpdbdpklpmppnoibabdkkhnfhkkehgnc
Firefox: https://addons.mozilla.org/en-US/firefox/addon/nbgitpuller-link-generator/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search